public/app/features/browse-dashboards/components/NameCell.tsx

```diff
@@ -59,43 +52,36 @@ export function NameCell({ row: { original: data }, onFolderClick }: NameCellPro
       <Indent level={level} />
 
       {item.kind === 'folder' ? (
-        <>
-          {isLoading ? (
-            <Spinner className={styles.chevron} />
-          ) : (
-            <IconButton
-              size={CHEVRON_SIZE}
-              className={styles.chevron}
-              ref={chevronRef}
-              onClick={() => {
-                if (!isOpen && !childrenByParentUID[item.uid]) {
-                  setShouldRestoreFocus(true);
-                }
-                onFolderClick(item.uid, !isOpen);
-              }}
-              name={isOpen ? 'angle-down' : 'angle-right'}
-              tooltip={isOpen ? 'Collapse folder' : 'Expand folder'}
-            />
-          )}
-        </>
+        <IconButton
+          size={CHEVRON_SIZE}
+          className={styles.chevron}
+          onClick={() => {
+            onFolderClick(item.uid, !isOpen);
+          }}
+          name={isOpen ? 'angle-down' : 'angle-right'}
+          aria-label={isOpen ? 'Collapse folder' : 'Expand folder'}
+        />
       ) : (
         <span className={styles.folderButtonSpacer} />
       )}
-      <Span variant="body" truncate>
-        {item.url ? (
-          <Link
-            onClick={() => {
-              reportInteraction('manage_dashboards_result_clicked');
-            }}
-            href={item.url}
-            className={styles.link}
-          >
-            {item.title}
-          </Link>
-        ) : (
-          item.title
-        )}
-      </Span>
+      <div className={styles.iconNameContainer}>
```

Comment:
would've loved to use `Stack` here, but there's no way of setting `overflow: 'hidden'` on a `Stack` so the text truncation doesn't work with it...